DTS-STN/Alpha-Site
# Add missing email

# Description

[#547 DEV: Add missing email address to step 2/3 in sign up](https://jira-dev.bdm-dev.dts-stn.com/browse/SCL-132)

Last PR didn't fix the issue as the Dockerfile needed to be updated as well, which this PR addresses.

## Test Instructions

1. Build the app in a Docker container with the `--build-arg NEXT_PUBLIC_THANK_YOU_EMAIL=experience@servicecanada.gc.ca` flag
2.  Run the application and navigate to `/thankyou` to verify the email is showing

## Checklist


## Product and Sprint Backlog

[Jira](https://jira-dev.bdm-dev.dts-stn.com/secure/RapidBoard.jspa?rapidView=96)
